docs/source/whatsnew.rst

```diff
@@ -1,126 +1,190 @@
 
 Using the new chaincode lifecycle
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Use the following tutorials to get started with the new chaincode lifecycle:
-
-* :doc:`chaincode4noah`:
-  Provides a detailed overview of the steps required to install and define a
-  chaincode, as well as the capabilities available with the new model.
-
-* :doc:`build_network`:
-  If you want to start using the new lifecycle right away, the BYFN tutorial has
-  been updated to use the :doc:`commands/peerlifecycle` CLI to install and
-  define chaincode on a sample network.
-
-* :doc:`private_data_tutorial`:
-  Has been updated to demonstrate how to use :doc:`private-data/private-data`
-  collections with the new chaincode lifecycle.
-
-* :doc:`endorsement-policies`:
-  Learn how the new lifecycle allows you to use policies in the channel
-  configuration as chaincode endorsement policies.
-
-
-Alpine images
--------------
-
-Starting with v2.0, Hyperledger Fabric Docker images will use Alpine Linux, a
-security-oriented, lightweight Linux distribution. This means that Docker images
-are now much smaller, providing faster download and startup
-times, as well as taking up less disk space on host systems. Alpine Linux
-is designed from the ground up with security in mind, and the
-minimalist nature of the Alpine distribution greatly reduces the risk of
-security vulnerabilities.
+For existing Fabric deployments, you can continue to use the prior chaincode
+lifecycle with Fabric v2.0. The new chaincode lifecycle will become effective
+only when the channel application capability is updated to v2.0.
+See the Chaincode for Operators tutorial for complete details of the new
+chaincode lifecycle.
+
+External chaincode launcher
+---------------------------
+
+While chaincode is still run in a docker container by default in Fabric v2.0,
+the external chaincode launcher feature empowers operators to build and launch
+chaincode with the technology of their choice.
+
+* **Eliminate Docker daemon dependency** Prior releases of Fabric required
+  peers to have access to a Docker daemon in order to build and launch
+  chaincode - something that may not be desirable in production environments
+  due to the privileges required by the peer process.
+
+* **Alternatives to containers** Chaincode is no longer required to be run
+  in Docker containers, and may be executed in the operator’s choice of
+  environment (including containers).
+
+* **External builder executables** An operator can provide a set of external
+  builder executables to override how the peer builds and launches chaincode.
+
+* **Chaincode as an external service** Traditionally, chaincodes are launched
+  by and then connect to the peer. It is now possible to run chaincode as
+  an external service, for example in a Kubernetes pod, which a peer can
+  connect to and utilize for chaincode execution.
+
+See the external chaincode documentation to learn more about the
+external chaincode launcher feature.
+
```

Comment:
If the chaincode as an external service makes it in, we should add an entry for it here.

Reply:
lines 93 to 96 mention chaincode as an external service.